selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ConnectionsManagement.java

```diff
@@ -143,9 +149,8 @@ public void acceptAConnection(String username, String... fullName) {
       searchPeople(fullName[0], null, null, null);
     else
       searchPeople(username, null, null, null);
-    evt.waitForAndGetElement(ELEMENT_CONNECTION_CONFIRM_BTN.replace("${user}", username));
-    evt.clickByJavascript(ELEMENT_CONNECTION_CONFIRM_BTN.replace("${user}", username));
-    evt.waitForElementNotPresent(ELEMENT_CONNECTION_CONFIRM_BTN.replace("${user}", username));
+    $(byText("Confirm")).click();
+    $(byText("Confirm")).waitUntil(Condition.disappears, Configuration.timeout);
     info("Accepted to the user");
```

Comment:
Please use a variable in SocialLocator instead of using string, to be compliant with other tests.